### PR TITLE
Embarc 486/restyle fe

### PR DESCRIPTION
**Pull request by KristenWAlthoff**
To test:
- make a copy of the .env-example file
- copy and paste in an ARC_API_CLIENT_ID and ARC_API_CLIENT_SECRET from a production account
- npm start and appreciate the lovelier Arc-ier UI

### CODE REVIEW

File: src/styles.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export const StyledButton = styled.button`
+  background: rgb(0, 64, 55);
+  border: 5px;
+  color: #ffffff;
+  padding: 16px;
+  border-radius: 5px;
+  font-family: "DM Sans", sans-serif;
+  font-weight: bold;
+  font-size: 16px;
+  cursor: pointer;
```
Comment (collinclarke) [inline src/styles.jsx:21]:
```suggestion
  font-size: 16px;
  margin-top: 16px;
  cursor: pointer;
```

Comment (collinclarke) [reply src/styles.jsx:21]: I think this makes the type lockup better particular in the success/failure messaging

**Review by collinclarke — APPROVED** [review]
Awesome 🆒

File: public/index.html [hunk 2]
```diff
@@ -3,9 +3,15 @@
...
     <meta charset="utf-8" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" sizes="any">
-    <link rel="icon" href="%PUBLIC_URL%/icon.svg" type="image/svg+xml">
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon.png">
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" sizes="any" />
+    <link rel="icon" href="%PUBLIC_URL%/icon.svg" type="image/svg+xml" />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/apple-touch-icon.png" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```
[the remark this answers is not in the document]
Comment (arapoport3) [reply public/index.html:10]: neat i also had never heard of this

**Review by arapoport3 — APPROVED** [review]
nice!

**Pull request merged**